br/pkg/restore/data.go

```diff
@@ -120,8 +118,8 @@ func (recovery *Recovery) newRecoveryClient(ctx context.Context, storeAddr strin
 		grpc.FailOnNonTempDialError(true),
 		grpc.WithConnectParams(grpc.ConnectParams{Backoff: bfConf}),
 		grpc.WithKeepaliveParams(keepalive.ClientParameters{
-			Time:    time.Duration(10) * time.Second,
-			Timeout: time.Duration(300) * time.Second,
+			Time:    time.Duration(3) * time.Second,
+			Timeout: time.Duration(10) * time.Second,
```

Comment:
to `cfg.GRPCKeepaliveTime` and `cfg.GRPCKeepaliveTimeout`

Reply:
should be good idea

Reply:
fixed